cdap-api/src/main/java/co/cask/cdap/api/common/HttpErrorStatusProvider.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.common;
+
+/**
+ * CDAP HTTP Response Exception
+ */
+public interface HttpErrorStatusProvider {
+
+  /**
+   * Gives the HTTP status code for this exception if one needs to be returned. All exception classes which needs to
```

Comment:
There is nothing related to exception in this interface. Shouldn't talk about it.